src/include/stir/DiscretisedDensity.h

```diff
@@ -41,6 +41,7 @@
 #include "stir/CartesianCoordinate3D.h"
 #include "stir/Array.h"
 #include "stir/IO/ExamData.h"
+#include "stir/ProjDataInfo.h"
```

Comment:
shouldn't be needed if it wasn't before (will need to be in cxx, but probably already there)